pom.xml

```diff
@@ -43,7 +43,7 @@
         <formatterConfigPath>formatter.xml</formatterConfigPath>
         <docker.maven.plugin.version>0.40.2</docker.maven.plugin.version>
         <java.module.version>1.2.10</java.module.version>
-        <file.connector.version>2.0.0-SNAPSHOT</file.connector.version>
+        <file.connector.version>1.4.1</file.connector.version>
```

Comment:
The last version for the file-connector is 1.5.0
https://github.com/mulesoft/mule-file-connector/tags